tools/README.md

```diff
@@ -27,16 +23,15 @@ to test.
 
 The script `prepare_for_prebuilt_workers.go` takes the following options:
 * `-l `<br> Language and GITREF to benchmark. The language and its specific 
-GITREF wish to build workers from can be specified as `language:gitref`.
+GITREF wish to build workers from can be specified as `language:COMMIT_SHA`.
 May be repeated.
 * `-t` <br> Tag for prebuilt images. Tag is a required fields. Tag complies with 
 [docker tag's restrictions](https://docs.docker.com/engine/reference/commandline/tag/#extended-description). 
 * `-r` <br> Root directory of Dockerfiles.
 * `-build-only` <br> Option to build only, when specified 
-  `-build-only=true` images will only be built locally without being pushed to an 
-  image registry.
-* `-p` <br> Image registry to store images. In the example, the location of the
-images are:
+  `-build-only=true`, images will only be built locally.
+* `-p` <br> Image registry to store images. The helper script only support pushing images to GCR. If another image registry is chosen, the images could still be built through script with `-build-only=true`, the user could then push the images manually. The Dockerfiles that the script building from is avalible in [test-infra/containers/pre_built_images](test-infra/containers/pre_built_images). If using Google cloud registry, the image prefix should be in form of `gcr.io/project-name/your-directory-name`. In the example, the built images would be stored as: 
+ :
   ```
   gcr.io/grpc-testing/project-name/pre_built_workers/cxx:user-specified-tag
   gcr.io/grpc-testing/project-name/pre_built_workers/go:user-specified-tag
```

Comment:
This flow is a bit confusing. Let's try to break it up a bit. 

```suggestion
* `-p` <br> Image registry to store images. The image prefix should be in form of `gcr.io/project-name/your-directory-name` for the Google Container Registry. For example, the built images would be stored as `gcr.io/grpc-testing/project-name/pre_built_workers/cxx:user-specified-tag` and `gcr.io/grpc-testing/project-name/pre_built_workers/go:user-specified-tag`. 
   
   If using a registry other than GCR, the images should be built through the script with the flag `-build-only=true`. The user could then push the images manually.
   
   The Dockerfiles that the script uses to build are available in [test-infra/containers/pre_built_images](test-infra/containers/pre_built_images).
```

As for paragraph 2, "user could the push the images manually", this paragraph may be omitted. If the command is `docker push` and they set the image prefix to the appropriate path, it will work for any registry. For example,

- Dockerhub: set prefix to "username/" then `docker push username/image` will work.

- AWS registries: set prefix to their structure `aws-account-id.dkr.ecr.region.amazonaws.com/image` then `docker push aws-account-id.dkr.ecr.region.amazonaws.com/image` works the same way. So does Microsoft Azure registries, I believe.

Unless we revert to gcloud commands, there shouldn't be a need to make them perform extra steps.

Reply:
If we need multiple paragraphs, I'd remove the `<br>` statements. We can hold the flags for emphasis. HTML tags in markdown aren't the cleanest. It's generally best to avoid them whenever possible.

Reply:
The <br> is very useful for legibility. Please keep it.

Reply:
I think it is because I update README from UI, it doesn't allow me to modify it from remote client. I am closing this PR and open a new one.